end-to-end-tests/src/test/java/com/amazonaws/blox/CreateDaemonEnvironmentTest.java

```diff
@@ -58,6 +60,24 @@ public void tearDown() {
                 .environmentName(environmentName));
     // Cleanup ECS tasks
     stack.reset();
+
+    // Describe the environment again, should be ResourceNotFoundException
+    Throwable thrown =
```

Comment:
This should really be in a testcase, and not in the environment teardown. Otherwise all tests will fail in teardown if this is broken.

Reply:
It due to the delete-environment is part of the teardown.  When I was thinking about the delete-environment step, I put it into the teardown instead of inside the testcase, because it can try to cleanup the environment even   the test case fails in the middle.

Since we have verification for the delete-environment, it makes sense to be a part of the test case.   Even we potentially may lack data in dyanmodb if test case fail, but we use UUID for environment ID, it should be good for now.